files/build_templates/docker_image_ctl.j2

```diff
@@ -179,7 +179,15 @@ start() {
     # Obtain our HWSKU as we will mount directories with these names in each docker
     HWSKU=${HWSKU:-`$SONIC_CFGGEN -d -v 'DEVICE_METADATA["localhost"]["hwsku"]'`}
     {%- endif %}
-
+    {%- if docker_container_name == "database" %}
+    # Ensure that database_global.json file exists in database docker for multi-asic platform.
+    # If it does not exist, remove the current docker so that the file is created.
+    # This is mainly to ensure that database_global.json file is created when converting
+    # single asic VS to multi-asic VS.
+    if [[ ($NUM_ASIC -gt 1) && (-z $DEV) && (! -f "/var/run/redis/sonic-db/database_global.json") ]]; then
```

Comment:
Can we used the  constant SONIC_DB_GLOBAL_JSON instead of the filename. SONIC_DB_GLOBAL_JSON used in other places in this file

Reply:
Updated as suggessted.